Zend/zend_vm_def.h

```diff
@@ -9476,6 +9476,20 @@ ZEND_VM_HANDLER(202, ZEND_CALLABLE_CONVERT, UNUSED, UNUSED)
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(203, ZEND_BIND_SELF_REFERENCE, CV, UNUSED, REF)
+{
+	USE_OPLINE
+	zval *variable_ptr;
+
+	variable_ptr = GET_OP1_ZVAL_PTR_PTR_UNDEF(BP_VAR_W);
+
+	zend_closure_from_frame(variable_ptr, execute_data);
```

Comment:
It returns fake closure with `$closure->call($class, 3);`, which causes segfault when variable is accessed. I haven't figured it out how to deal with this yet.